duckduckgo/macos-browser
# downloads settings fix

Task/Issue URL: https://app.asana.com/0/392891325557410/1202355163274007
Tech Design URL:
CC:

**Description**:

Always read/write from the backing store since different instances of this class are used in both the UI and in the download manager.

**Steps to test this PR**:
1. Download the file from https://file-examples.com/wp-content/uploads/2017/02/zip_2MB.zip
1. Change the settings:
   * Ask where to save files
   * Download location
1. Ensure the settings are respected without having to restart the browser

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
